src/client/Client.js

```diff
@@ -337,6 +343,36 @@ export default class Client {
         return this;
     }
 
+    /**
+     * @param {number} minBackoff
+     */
+    setMinBackoff(minBackoff) {
+        this._minBackoff = minBackoff;
```

Comment:
This should do the same validations as `Executable.setMinBackoff()`. Right now this allows setting minBackoff to be greater than maxBackoff.